src/web/server/locales/en/common.json

```diff
@@ -26,7 +26,9 @@
   },
   "areYouPregnant": {
     "title": "Are you pregnant?",
-    "heading": "Are you pregnant?"
+    "heading": "Are you pregnant?",
+    "expectedDateOfDeliveryText": "When is your baby due to be born?",
+    "expectedDateOfDeliveryHint": "For example, 31 3 2019"
```

Comment:
Weren't we going to be making this always 6 months in the future?

Reply:
Yes .. There is another sub-task for this.